zebedee-reader/src/main/java/com/github/onsdigital/zebedee/reader/CollectionReader.java

```diff
@@ -59,7 +60,11 @@ public Page getContent(String path) throws ZebedeeException, IOException {
 
 
     public Resource getResource(String path) throws ZebedeeException, IOException {
-        return findResource(path);
+        try {
+            return findResource(path);
+        } catch (BadRequestException e) {
+            return findResource(path + "/index.html");
```

Comment:
Do we want to check if the uri starts with /visualisations? to limit what this will be applied to? Also what happens if there is no index.html? do we get a 404?